subsys/bluetooth/services/dis.c

```diff
@@ -38,6 +47,48 @@ static struct dis_pnp dis_pnp_id = {
 };
 #endif
 
+#ifdef CONFIG_BT_GATT_DIS_SETTINGS
```

Comment:
minor: we try to be consistent and use `if defined(CONFIG_BT_...)` throughout the Bluetooth subsystem. It doesn't make much difference for a single condition, but for multiple ones it has the benefit that you can chain them with `||` or `&&` (something you can't do with `#ifdef CONFIG_BT...`).

Reply:
OK - Changed - Did not know.